src/Compilers/CSharp/Portable/Symbols/Synthesized/Records/SynthesizedRecordEquals.cs

```diff
@@ -13,17 +13,48 @@
 
 namespace Microsoft.CodeAnalysis.CSharp.Symbols
 {
+    /// <summary>
+    /// A strongly-typed `public bool Equals(T other)` method.
+    /// There are two types of strongly-typed Equals methods:
+    /// the strongly-typed virtual method where T is the containing type; and
+    /// overrides of the strongly-typed virtual methods from base record types.
```

Comment:
>overrides of the strongly-typed virtual methods from base record types [](start = 8, length = 70)

According to the specification at [https://github.com/dotnet/csharplang/blob/master/proposals/records.md](https://github.com/dotnet/csharplang/blob/master/proposals/records.md) no such overrides are supposed to be produced.